odoo/addons/base/models/ir_model.py

```diff
@@ -1567,6 +1557,18 @@ def _update_xmlids(self, data_list, update=False):
         # update loaded_xmlids
         self.pool.loaded_xmlids.update("%s.%s" % row[:2] for row in rows)
 
+    def _build_update_xmlids_query(self, sub_rows, update):
+        rowf = "(%s, %s, %s, %s, %s, now() at time zone 'UTC', now() at time zone 'UTC')"
```

Comment:
the more complex version of this would have build the query piece by piece, so that if another module than studio needs to override it, they could without having deps problems.

this would pbly be quite complex for something that may never happen, so instead can i just ask you to add this comment here to avoid something trowing this away too easily?
```py
# NOTE: this method is overriden in web_studio; if you need to make another override, make sure it is compatible with the one that is there.
```

Reply:
Yes. The comment has been added.